codec/src/test/java/io/netty/handler/codec/marshalling/AbstractMarshallingTest.java

```diff
@@ -37,6 +38,6 @@ public static void checkSupported() throws Throwable {
             }
             error = cause;
         }
-        Assume.assumeNoException(error);
+        assumeTrue(error == null, error + " was not null");
```

Comment:
There is no replacement for `assumeNoException`, so I *assumed* (ha!) this should work - a second look might be good here.

Reply:
I think this is fine :)